pandas/tools/plotting.py

```diff
@@ -1217,8 +1217,25 @@ def _adorn_subplots(self):
 
         if self.title:
             if self.subplots:
-                self.fig.suptitle(self.title)
+                if is_list_like(self.title):
+                    if len(self.title) != self.nseries:
+                        msg = 'The length of `title` must equal the number ' \
+                              'of columns if using `title` of type `list` ' \
+                              'and `subplots=True`.\n' \
+                              'length of title = {}\n' \
```

Comment:
Can you use parentheses around the full string instead of `\` for the line continuation? (for consistency within the project)